### ISSUE TITLE
get_structure() for sphinx jobs gives error

### ISSUE THREAD

**Issue by skatnagallu** [pyiron/pyiron_atomistics#1378]
**Summary**

running get_structure() for sphinx job gives error (IndexError: ('cell at step ', 25, ' not found')).  Sphinx Job is a DFT relaxation job. and cell typically doesn't change for each step.

```python
job = pr.load('sphinx_job')
job.get_structure(i) # i>0 gives the error.
```

**Comment by samwaseda** [comment]
I should open a PR or are you going to work on it?

**Comment by skatnagallu** [comment]
@samwaseda 
I wanted to confirm before I made any changes. The only part in sphinx base, where cells are written to output is [here](https://github.com/pyiron/pyiron_atomistics/blob/b6f580780640f359c05df74b75b89860c88c225e/pyiron_atomistics/atomistics/job/atomistic.py#L1012) 
```python
        if "cells" not in self.generic.list_nodes():
            self.generic.cells = np.array([self._job.structure.cell.tolist()])
```

So as you said  changing this line to 
```python
        if "cells" not in self.generic.list_nodes():
            self.generic.cells = np.tile(np.array([self._job.structure.cell.tolist()]),(len(self.generic.energy_tot),1,1))
```
should work?

**Comment by samwaseda** [comment]
Yeah I guess this would work. But this has to be implemented in SPHInX base

**Comment by skatnagallu** [comment]
But sphinx base output class has no property of cells. should I then create it?

**Comment by samwaseda** [comment]
Yes

**Comment by skatnagallu** [comment]
The issue still exists. As @freyso and I discussed today, the `get_structure()` method tries to get the cell information from the output level, where as the modifications we did work on `output.generic` level.  So I guess we could debate at which level the methods should access the data? And how do we fix this?
